docs/src/design-system/typography/TextStylesPage.tsx

```diff
@@ -74,7 +75,7 @@ const TextStylesPage: React.FC = () => {
                     key={column}
                     align="left"
                     sx={{
-                      ...tableCellStyles(palette),
+                      ...tableCellStyles,
```

Comment:
You know, you can do this just fine with `sx={tableCellStyles}`, but if you prefer the way you have it with the spread, that's fine - it does have the advantage of being able to easily add on other styles if needed.

Reply:
Thanks for pointing it out!